src/data/camouflages.js

```diff
@@ -61,7 +61,10 @@ export default [
       'Pistols': { default: '25 Longshots' },
       'Shotguns': { default: '50 Longshots' },
       'Sniper Rifles': { default: '50 Longshots' },
-      'Special': { default: 'Destroy 50 ground-based killstreaks' },
+      'Special': { 
+        default: 'Destroy 50 killstreaks or equipment',
+        'R1 Shadowhunter': '50 Longshots'
```

Comment:
Shouldn't this be "50 Headshots" according to #15?

Reply:
Fixed!